spec/parser/arguments_spec.rb

```diff
@@ -40,6 +40,12 @@ def parse(*args)
       expect(parse("product", "title", "age")["array"]).to eq(%w(title age))
     end
 
+    it "accepts - as an array argument" do
+      create :array => nil
+      expect(parse("-")["array"]).to eq(%w(-))
+      expect(parse("-", "title", "-")["array"]).to eq(%w(- title -))
+    end
```

Comment:
I ran this test against master and, as expected, it fails. I don't think there's any other cases we need to consider for this feature.

I also went through the docs and feel like we don't need to update/change anything: IMO this change is to allow functionality that should already (implicitly) be present, not a new feature.